sugar-config.cmake.in

```diff
@@ -0,0 +1,4 @@
+@PACKAGE_INIT@
+
+include(cmake/Sugar)
```

Comment:
I've tried to use the package but got error:
```
share/sugar/sugar-config.cmake:27 (include):
  include could not find load file:

    cmake/Sugar
```

I guess the solution is to use full path:
```cmake
include("${CMAKE_CURRENT_LIST_DIR}/cmake/Sugar")
```